#! /bin/sh /usr/share/dpatch/dpatch-run
## wvtext-no-graphics.dpatch by Matej Vela <vela@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: wvText.in: Disable graphics output in wvText.xml conversion
## DP: (already disabled in wvHtml conversion).

@DPATCH@
diff -urNad wv-1.2.1~/wvText.in wv-1.2.1/wvText.in
--- wv-1.2.1~/wvText.in	2003-11-03 18:03:14.000000000 +0100
+++ wv-1.2.1/wvText.in	2006-03-18 11:15:17.000000000 +0100
@@ -78,5 +78,5 @@
     # fall back onto our cruddy output
     # this is, admittedly, better than running
     # 'strings' on the word document though :)
-    wvWare -x ${datadir}/wv/wvText.xml "${1}" > "${2}"
+    wvWare -1 -x ${datadir}/wv/wvText.xml "${1}" > "${2}"
 fi
